ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/MyJobsFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Switch } from '@patternfly/react-core';
+
+export type MyJobsFilterProps = {
+    showOnlyMyJobs: boolean;
+    onToggle: (checked: boolean) => void;
```

Comment:
Because `Toggle` is a different PatternFly element, what do you think about alternative name?
Although verbose, either `onChangeMyJobsFilter` or `onMyJobsFilterChange` are starting points for brainstorming, both here and for callback function in the next file.

Reply:
Fair point. That callback function could be clearer. Do you think `onToggleMyJobs` is less verbose but still a good description?